install/gcp/installer/forseti_client_installer.py

```diff
@@ -87,6 +87,6 @@ def get_deployment_values(self):
             'FORSETI_BUCKET': bucket_name[len('gs://'):],
             'BUCKET_LOCATION': self.config.bucket_location,
             'GCP_CLIENT_SERVICE_ACCOUNT': self.gcp_service_acct_email,
-            'BRANCH_OR_RELEASE': 'branch-name: "{}"'.format(self.branch),
+            'FORSETI_TARGET': 'forseti-target: "{}"'.format(self.branch),
```

Comment:
Now that I see this here, is there any reason why `forseti-target` attribute is defined here, rather in the yaml.in files?

Reply:
No particular reason, it has been like that before. Moved the variable to the yaml file.